src/App.jsx

```diff
@@ -14,57 +15,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const clearButton = () => {
+    setSelectedGood('');
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood && `${selectedGood} is selected`}
+        {selectedGood ? (
+          <button
+            onClick={clearButton}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        ) : (
+          'No goods selected'
+        )}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
+      <table className="table">
+        <tbody>
+          {goods.map(goodName => (
+            <tr
+              data-cy="Good"
+              key={goodName}
+              className={
+                selectedGood === goodName && 'has-background-success-light'
+              }
             >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+              <td>
+                {selectedGood !== goodName ? (
+                  <button
+                    data-cy="AddButton"
+                    type="button"
+                    className={`button ${goodName === selectedGood && 'has-background-success-light'}`}
+                    onClick={() => setSelectedGood(goodName)}
+                  >
+                    +
+                  </button>
+                ) : (
+                  <button
+                    data-cy="RemoveButton"
+                    type="button"
+                    className="button is-info"
+                    onClick={clearButton}
+                  >
+                    -
+                  </button>
+                )}
```

Comment:
Please, try using single `button` component and alter it's props, not to copypatse the code

```suggestion
                {selectedGood !== goodName ? (
<button
   data-cy={selectedGood !== goodName  ? "AddButton" : "RemoveButton"}
   type="button"
   className={`button ${goodName === selectedGood && 'has-background-success-light'}`}
   onClick={() => setSelectedGood(selectedGood !== goodName ? goodName : '')}
>
       {...}
</button>
```